scripts/bootstrap.sh.in

```diff
@@ -150,8 +151,20 @@ orchestrate_bootstrap() {
         "$SALT_CALL" mine.update
 }
 
+
+bootstrap_file_is_present() {
+    if [ ! -f "$CONFIG_FILEPATH" ]; then
+        die "You need to create a BootstrapConfiguration file, at $CONFIG_FILEPATH." \
+            "Please refer to the Installation Guide here:" \
+            "https://metal-k8s.readthedocs.io/en/latest/installation/bootstrap.html#configuration" >&2
```

Comment:
I wouldn't put a URL here. The `latest` link may not be the 'right' one. Refer to the documentation in a more general sense, and if need be to the PDF on the ISO.